server/lib/template/helpers/dates.js

```diff
@@ -17,7 +17,7 @@ const TIMESTAMP_FMT = 'DD/MM/YYYY HH:mm:ss';
  * @returns {String} - the formatted string for insertion into templates
  */
 function date(value, dateFormat) {
-  dateFormat = dateFormat.name === 'date' ? DATE_FMT : dateFormat;
+  dateFormat = (!dateFormat || dateFormat.name === 'date') ? DATE_FMT : dateFormat;
```

Comment:
This is a bit confusing to read with `!` and `||`.  I think this is a better way of expressing the same thing:
```js
dateFormat = (dateFormat && dateFormat.name !== 'date') || DATE_FMT;
```
Then it's clear to me what is default.  What do you think?